osquery/tables/system/windows/logged_in_users.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#define _WIN32_DCOM
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Wtsapi32.h>
+#include <winsock2.h>
+
+#include <osquery/core.h>
+#include <osquery/core/conversions.h>
```

Comment:
Use the `"osquery/core/conversions.h"` local search method in a block of includes below the public APIs.